build_projects/Microsoft.DotNet.Cli.Build.Framework/CurrentPlatform.cs

```diff
@@ -78,10 +87,15 @@ public static bool IsLinux
         {
             get
             {
-                return IsUbuntu || IsCentOS || IsRHEL || IsDebian;
+                return IsUbuntu || IsCentOS || IsRHEL || IsDebian || IsFedora;
             }
         }
 
+        public static bool IsPlatform(BuildPlatform platform, string version = null)
+        {
+            return IsPlatform(platform) && IsVersion(version);
```

Comment:
Once you that, you need to make a pass over existing things conditional on ubuntu and decide if you need to add version numbers.